.github/workflows/continuous-integration-workflow.yml

```diff
@@ -0,0 +1,21 @@
+name: Continuous Integration
+on: 
+   push:                    # Will make the workflow run every time you push to any branch
+    paths-ignore:
+    - '**/README.md'
+   pull_request:
+    paths-ignore:
+    - '**/README.md'
+
+jobs: 
+  build:
+    name: Build and test
+    runs-on: ubuntu-latest     # Sets the build environment a machine with the latest Ubuntu installed
+    steps:
+    - uses: actions/checkout@v2  # Adds a step to checkout the repository code
+    
+    - name: Build Docker
+      run: docker build --tag todo-app:devtest --target devtestpy .
+
+    - name: Run Docker
+      run: docker run -e FLASK_APP=${{ secrets.FLASK_APP_X}} -e FLASK_ENV=${{ secrets.FLASK_ENV_X}} -e SECRET_KEY=${{ secrets.SECRET_KEY_X }} todo-app:devtest
```

Comment:
Similarly to a comment below - this suggests you're using real credentials for your tests, but your tests only need fake ones! Using real ones _shouldn't_ cause any problems but it does increase the risk that you accidentally make genuine requests to Trello, so I'd suggest switching this to use `--env-file .env.test` instead